src/widgets/voice-search/voice-search.tsx

```diff
@@ -61,7 +63,7 @@ const renderer: VoiceSearchRenderer<VoiceSearchRendererWidgetParams> = ({
 
   render(
     <VoiceSearch
-      cssClasses={cssClasses}
+      cssClasses={cssClasses as VoiceSearchComponentCSSClasses}
```

Comment:
You don't need to cast `cssClasses` here if you change the type of `VoiceSearchRendererWidgetParams['cssClasses']` to `VoiceSearchComponentCSSClasses`.

Reply:
@francoischalifour If I don't cast it, it doesn't pass `yarn type-check`.

![image](https://user-images.githubusercontent.com/499898/58018072-1c82dc80-7b02-11e9-9b95-8f3e4e5b94b3.png)

Reply:
It should work if you change [this line](https://github.com/algolia/instantsearch.js/blob/c4f3e5ba83aef2cf63de53a1034ae8ad13fdab11/src/widgets/voice-search/voice-search.tsx#L49):

```diff
interface VoiceSearchRendererWidgetParams extends VoiceSearchWidgetParams {
  container: HTMLElement;
- cssClasses: VoiceSearchCSSClasses;
+ cssClasses: VoiceSearchComponentCSSClasses;
  templates: VoiceSearchTemplates;
}
```

Reply:
Right. That's where I can change the type! I was lost in params hell. Thanks for saving me.